OIPA/iati/tests/test_parser_post_models.py

```diff
@@ -233,11 +237,11 @@ def test_set_sector_transaction_without_percentages(self):
 
         ts1 = TransactionSector.objects.filter(sector=self.s1, transaction=self.t1)[0]
         # 10000 / 2
-        self.assertEqual(ts1.percentage, 50)
+        self.assertEqual(ts1.percentage, 100)
 
         ts2 = TransactionSector.objects.filter(sector=self.s2,transaction=self.t2)[0]
         # 20000 / 3
-        self.assertEqual(round(ts2.percentage), 50)
+        self.assertEqual(round(ts2.percentage), 100)
 
```

Comment:
The actual code looks good imo. I'd rather expand the test a bit to be sure:

**setUpCountriesRegionsSectors**
create 4 sectors, of which 2 for each vocabulary.

**test_set_sector_transaction_without_percentages**
Split into 2 tests, first one should test voc 1, other voc 2. Within the current set-up I'm not afraid of current functionality, but if we would change the functionality later and that has the implication of settings all transactionsectors to 100, we would not discover it with the unit test above.